jvm-runtime/testdata/java/javaclient/src/main/java/xyz/block/ftl/test/TestInvokeGoFromJava.java

```diff
@@ -59,176 +59,176 @@ public class TestInvokeGoFromJava {
-        return client.call(val);
+        return client.objectArrayVerb(val);
     }
 
     @Export
     @Verb
     public @NotNull ParameterizedType<String> parameterizedObjectVerb(@NotNull ParameterizedType<String> val,
             ParameterizedObjectVerbClient client) {
-        return client.call(val);
+        return client.parameterizedObjectVerb(val);
     }
 
     @Export
     @Verb
     public @NotNull ZonedDateTime timeVerb(@NotNull ZonedDateTime instant, TimeVerbClient client) {
-        return client.call(instant);
+        return client.timeVerb(instant);
     }
 
     @Export
     @Verb
     public @NotNull TestObject testObjectVerb(@NotNull TestObject val, TestObjectVerbClient client) {
-        return client.call(val);
+        return client.testObjectVerb(val);
     }
 
     @Export
     @Verb
     public @NotNull TestObjectOptionalFields testObjectOptionalFieldsVerb(@NotNull TestObjectOptionalFields val,
             TestObjectOptionalFieldsVerbClient client) {
-        return client.call(val);
+        return client.testObjectOptionalFieldsVerb(val);
     }
 
     // now the same again but with option return / input types
 
     @Export
     @Verb
     public Long optionalIntVerb(Long val, OptionalIntVerbClient client) {
-        return client.call(val);
+        return client.optionalIntVerb(val);
     }
 
     @Export
     @Verb
     public Double optionalFloatVerb(Double val, OptionalFloatVerbClient client) {
-        return client.call(val);
+        return client.optionalFloatVerb(val);
     }
 
     @Export
     @Verb
     public @Nullable String optionalStringVerb(@Nullable String val, OptionalStringVerbClient client) {
-        return client.call(val);
+        return client.optionalStringVerb(val);
     }
 
     @Export
     @Verb
     public byte @Nullable [] optionalBytesVerb(byte @Nullable [] val, OptionalBytesVerbClient client) {
-        return client.call(val);
+        return client.optionalBytesVerb(val);
     }
 
     @Export
     @Verb
     public Boolean optionalBoolVerb(Boolean val, OptionalBoolVerbClient client) {
-        return client.call(val);
+        return client.optionalBoolVerb(val);
     }
 
     @Export
     @Verb
     public @Nullable List<String> optionalStringArrayVerb(@Nullable List<String> val, OptionalStringArrayVerbClient client) {
-        return client.call(val);
+        return client.optionalStringArrayVerb(val);
     }
 
     @Export
     @Verb
     public @Nullable Map<String, String> optionalStringMapVerb(@Nullable Map<String, String> val,
             OptionalStringMapVerbClient client) {
-        return client.call(val);
+        return client.optionalStringMapVerb(val);
     }
 
     @Export
     @Verb
     public @Nullable ZonedDateTime optionalTimeVerb(@Nullable ZonedDateTime instant, OptionalTimeVerbClient client) {
-        return client.call(instant);
+        return client.optionalTimeVerb(instant);
```

Comment:
I wonder if we should still stick with 'call' as the default name. You already have the name in the interface, so I don't think we need to duplicate the name.

Reply:
I was actually wondering if we should combine all verbs into a single interface/class, then we get an API like:
```
    @Export
    @Verb
    public @Nullable ZonedDateTime optionalTimeVerb(@Nullable ZonedDateTime instant, GoModule goModule) {
        return goModule.optionalTimeVerb(instant);
    }
```

Reply:
The issue is then you can't detect usages. I guess if you have a few that are commonly used you could allow a pattern where you have `CombinedInterface extends Client1, Client2`.

Reply:
Let's leave the names as-is for now, and see how it goes.

Reply:
Yeah, I hadn't considered the usage detection. Worth discussing all this next week